scripts/gengodoc.awk

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env -S awk -f
+$0 ~ "//\tUsage:" {
+        in_usage = 1
+}
+$0 ~ "^(package |// [[])" && in_usage {
+        system(cmd "| sed -e '/./s|^|//\t|' -e 's|^$|//|'")
```

Comment:
I've changed my mind about this - passing `cmd` via an arg. Instead this could be:

    system(cmd " --help | sed ...")

and put `cmd = $3` after `in_usage = 1` in the action above.